components/checkbox/stories/checkbox.stories.js

```diff
@@ -130,7 +133,153 @@ const CheckboxGroup = (args) => html`
 	</div>
 `;
 
+// AllVariantsCheckboxGroup displays the default, invalid, disabled and read-only variations as a single
```

Comment:
The "variant checkbox groups" are my attempt at showing the 4 variations in a single story. If there's a better way to do this, I can definitely refactor `AllVariantsCheckboxGroup`.